spec/ndt7-protocol.md

```diff
@@ -163,6 +163,15 @@ be implemented so that immediately after thirteen seconds have elapsed, the
 underlying TLS connection is closed. This can be implemented, e.g., in C/C++
 using alarm(3) to cause pending I/O operations to fail with `EINTR`.
 
+As regards ping and pong messages, a ndt7 server MAY send periodic
+ping messages during any test. Servers MUST NOT send ping messages more
+frequently than they would send textual messages. Clients SHOULD be prepared
+to receive ping messages. They SHOULD reply to such messages with pong
+messages containing the same payload. If a client is receiving too
+many ping messages in a specific time interval, it MAY drop them at
+random. A client MAY decide to ignore ping messages. Clients that
```

Comment:
Ignoring PING is contrary to the websocket spec.  Let's not allow NDT7 clients to behave in ways which contradict the websocket RFC.

Reply:
https://tools.ietf.org/html/rfc6455#section-5.5.2 -    Upon receipt of a Ping frame, an endpoint MUST send a Pong frame in response, unless it already received a Close frame.  It SHOULD respond with Pong frame as soon as is practical.

Reply:
That text was meant to make half-duplex clients compliant during the upload. I am amending the text to omit that text. I believe it's reasonable to argue that, for an half-duplex client, as soon as practical means after it has finished uploading ;-). Do you want me to explicit that?

Reply:
See 0dc5a4f

Reply:
New language looks good to me.